pnpm-lock.yaml

```diff
@@ -30,7 +30,7 @@ importers:
       '@sveltejs/eslint-config': github.com/sveltejs/eslint-config/9a7d728e03ac433e5856a6e06775c17ee986d641_d9525e585486a10ae6317f3f61e9597f
```

Comment:
you shouldn't need to modify the lockfile. there's a merge conflict since this file is being changed, so it'd be nice to drop it from the PR

Reply:
Reverted to the previous lockfile. It appears the build is failing due to the new config options.

`Unexpected option config.kit.methodOverride`

I'm assuming Netlify is building the `create-svelte` apps using the version of sveltekit provided by npm, and not this version of the kit? How is this typically handled?